src/Sulu/Bundle/ResourceBundle/Api/Filter.php

```diff
@@ -0,0 +1,288 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ResourceBundle\Api;
+
+use Sulu\Bundle\ResourceBundle\Entity\FilterTranslation;
+use Sulu\Component\Rest\ApiWrapper;
+use Sulu\Bundle\ResourceBundle\Entity\Filter as FilterEntity;
+use Sulu\Bundle\ResourceBundle\Api\ConditionGroup;
+use JMS\Serializer\Annotation\VirtualProperty;
+use JMS\Serializer\Annotation\SerializedName;
+use JMS\Serializer\Annotation\ExclusionPolicy;
+use Hateoas\Configuration\Annotation\Relation;
+use Sulu\Component\Security\Authentication\UserInterface;
+
+/**
+ * The Filter class which will be exported to the API
+ *
+ * @package Sulu\Bundle\ResourceBundle\Api
+ * @Relation("self", href="expr('/admin/api/filters/' ~ object.getId())")
+ * @ExclusionPolicy("all")
+ */
+class Filter extends ApiWrapper
+{
+
+    /**
+     * @param FilterEntity $entity
+     * @param string $locale
+     */
+    public function __construct(FilterEntity $entity, $locale)
+    {
+        $this->entity = $entity;
+        $this->locale = $locale;
+    }
+
+    /**
+     * Get id
+     *
+     * @VirtualProperty
+     * @SerializedName("id")
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->entity->getId();
+    }
+
+    /**
+     * Returns the name of the filter
+     *
+     * @return string The name of the filter
+     * @VirtualProperty
+     * @SerializedName("name")
+     */
+    public function getName()
+    {
+        return $this->getTranslation()->getName();
+    }
+
+    /**
+     * Get andCombination
+     *
+     * @VirtualProperty
+     * @SerializedName("andCombination")
+     * @return boolean
+     */
+    public function getAndCombination()
```

Comment:
call this `conjunction` and write into `and` or `or` ... that it is possible to add more than that e.g. `xor` or something like that.